readme.md

```diff
@@ -179,6 +179,10 @@
 
 ## Resources
 
+### Guidelines
+
+- [Coding Guidelines for Prolog](https://arxiv.org/abs/0911.2899) - Coding standards and good practices that are fundamental in Prolog programming.
+
```

Comment:
It would probably be more fitting to add this at the bottom of the `Books` section, since it is a paper.
Also, consider a more compact description, for example:

- [Coding Guidelines for Prolog](https://arxiv.org/abs/0911.2899) - Coding standards and good practices for Prolog.